dev_tools/notebooks/notebook_test.py

```diff
@@ -45,8 +46,12 @@
 
 
 def _list_all_notebooks() -> Set[str]:
-    output = subprocess.check_output(['git', 'ls-files', '*.ipynb'])
-    return set(output.decode('utf-8').splitlines())
+    try:
+        output = subprocess.check_output(['git', 'ls-files', '*.ipynb'])
+        return set(output.decode('utf-8').splitlines())
+    except subprocess.CalledProcessError as ex:
+        warning("It seems that tests are run from not a git repo, notebook tests are skipped", ex)
```

Comment:
Why is the default behavior to skip on failure? Can we instead append the message to the error?